### PR TITLE
Increase split processing config parameters

### PR DESCRIPTION
**Pull request by Dith3r**
## Description

Improve query performance with easy to process splits by increasing the number of splits assigned, and makes initial worker split concurrency two times higher, but much slower to be adjusted (increased) based on memory utilization.

Salesforce benchmark:
image
[c_sales.pdf](https://github.com/starburstdata/galaxy-trino/files/13051858/c_sales.pdf)

Hive/parquet/part:
image
[c_hive_par_part.pdf](https://github.com/starburstdata/galaxy-trino/files/13051863/c_hive_par_part.pdf)

Hive/parquet/unpart:
image
[c_hive_par_unpart.pdf](https://github.com/starburstdata/galaxy-trino/files/13051866/c_hive_par_unpart.pdf)

Iceberg/parquet/part:
image
[c_iceberg_par_part.pdf](https://github.com/starburstdata/galaxy-trino/files/13051867/c_iceberg_par_part.pdf)

Iceberg/parquet/unpart:
image
[c_iceberg_par_unpart.pdf](https://github.com/starburstdata/galaxy-trino/files/13051868/c_iceberg_par_unpart.pdf)

## Release notes

( ) This is not user-visible or is docs only, and no release notes are required.
( ) Release notes are required. Please propose a release note for me.
(x) Release notes are required, with the following suggested text:

```markdown
# General
* Improve performance of queries with short running splits. ({issue}`19487`)
```

### CODE REVIEW

**Review by radek-starburst — APPROVED** [review]

**Review by raunaqmorarka — APPROVED** [review]
Please make sure that any documentation around these properties gets an update and add release notes entry to the PR description.

File: core/trino-main/src/main/java/io/trino/execution/TaskManagerConfig.java [hunk 1]
```diff
@@ -56,10 +56,10 @@ public class TaskManagerConfig
     private boolean shareIndexLoading;
     private int maxWorkerThreads = Runtime.getRuntime().availableProcessors() * 2;
     private Integer minDrivers;
-    private Integer initialSplitsPerNode;
+    private int initialSplitsPerNode = Runtime.getRuntime().availableProcessors() * 4;
     private int minDriversPerTask = 3;
     private int maxDriversPerTask = Integer.MAX_VALUE;
-    private Duration splitConcurrencyAdjustmentInterval = new Duration(100, TimeUnit.MILLISECONDS);
+    private Duration splitConcurrencyAdjustmentInterval = new Duration(1, TimeUnit.SECONDS);
```
Comment (raunaqmorarka) [inline core/trino-main/src/main/java/io/trino/execution/TaskManagerConfig.java:62]: Should this be a separate commit ?
Please add rationale for changing this config to the PR description. It's not super obvious how this relates to assigning more splits

Comment (Dith3r) [reply core/trino-main/src/main/java/io/trino/execution/TaskManagerConfig.java:62]: I will extend description.

Comment (Dith3r) [reply core/trino-main/src/main/java/io/trino/execution/TaskManagerConfig.java:62]: Initial worker split concurrency is two times higher, but much slower to be adjusted (increased) based on memory utilization.

Comment (sopel39) [reply core/trino-main/src/main/java/io/trino/execution/TaskManagerConfig.java:62]: > Initial worker split concurrency is two times higher, but much slower to be adjusted (increased) based on memory utilization.

It would be great to add a word why it helps. Even as top-level retro-post here in PR

Comment (Dith3r) [reply core/trino-main/src/main/java/io/trino/execution/TaskManagerConfig.java:62]: added to description

**Comment by xiacongling** [comment]
hi, @Dith3r . Links cannot be opened (http 404). Can you fix them (may need to make them public). Thanks!
